### ISSUE TITLE
Launcher?

### ISSUE THREAD

**Issue by NineKain** [ihm-tswow/Links-Awakening-DX-HD#15]
Would it be possible to create an updatable launcher, like most other PC ports/fan games? (AM2R, Zelda3, Ship of Harkinian, SM64), so you dont have to build the game every time a PR merges

It would improve the usability of this fan game a lot, since people aren't that tech savvy (like me)

**Comment by Requiett** [comment]
This should probably be filed as a Pull request.

**Comment by NineKain** [comment]
I'd love to if I knew how to code, it's a suggestion

**Comment by Requiett** [comment]
GitHub has a tab for Issues and a tab for Pull Requests. Generally requesting a new feature instead of addressing an existing one should be filed as a Pull Request. It's at the top of the page to the right of Issues.

**Comment by NineKain** [comment]
How can I add a PR without writing code...?

**Comment by Requiett** [comment]
It's a tab at the top of the GitHub page for the project. It works the same as the "Issues" section. It's not a matter of coding, it's a matter of organization for the project.

**Comment by Requiett** [comment]
![2023-12-26 12_16_37-Launcher_ · Issue #15 · ihm-tswow_Links-Awakening-DX-HD - Brave](https://github.com/ihm-tswow/Links-Awakening-DX-HD/assets/154869496/a87aa662-6fa2-4b71-9852-9a3f7a1d2a89)

**Comment by SirGouki** [comment]
You don't.  If you haven't written any code, there's nothing to pull.  A PR is for updating the repo with changed code or resources.

**Comment by NineKain** [comment]
Yeah thats what I tried to say, seems the guy deleted its comments

**Comment by filipe-maia** [comment]
This issue is **being solved** or is **solved** here:
https://github.com/BigheadSMZ/Links-Awakening-DX-HD

Please report more issues (if you find them) here:
https://github.com/BigheadSMZ/Links-Awakening-DX-HD/issues
For more information read:
https://github.com/ihm-tswow/Links-Awakening-DX-HD/issues/50
